### PR TITLE
Add bulk meter reading API for faster creation

### PR DESCRIPTION
**Pull request by nllong**
#### Any background context you want to provide?
Meter readers only supported one-at-a-time style import.

#### What's this PR do?
* Add in a Bulk Meter Reading Create

#### How should this be manually tested?
* Should be tested in part of the Meter Viewset PR
* API only updates

#### What are the relevant tickets?
N/A

### CODE REVIEW

File: seed/tests/test_meter_views.py [hunk 1]
```diff
@@ -202,6 +202,47 @@ def test_create_meter_readings(self):
...
+                "start_time": values[0],
+                "end_time": values[1],
+                "reading": values[2],
+                "source_unit": "Wh (Watt-hours)",
+                # conversion factor is required and is the conversion from the source unit to kBTU (1 Wh = 0.00341 kBtu)
+                "conversion_factor": 0.00341,
+            })
+
+        response = self.client.post(url, data=json.dumps(payload), content_type='application/json')
+        print(response)
```
Comment (haneslinger) [inline seed/tests/test_meter_views.py:236]: stray print

File: seed/serializers/meter_readings.py [hunk 2]
```diff
@@ -4,23 +4,39 @@
...
+    def create(self, validated_data):
+        print(validated_data)
+        meter_data = [MeterReading(**item) for item in validated_data]
+
+        try:
+            result = MeterReading.objects.bulk_create(meter_data)
+        except IntegrityError as e:
+            raise ValidationError(e)
+
+        return result
```
Comment (haneslinger) [inline seed/serializers/meter_readings.py:24]: I don't think I understand why we need both serializes... other seralizers seem able to hand bulk and singular input.

```
FilterGroupSerializer(filter_groups, many=True).data
```

**Review by haneslinger — APPROVED** [review]

File: seed/serializers/meter_readings.py (hunk 2 above)
Comment (nllong) [reply seed/serializers/meter_readings.py:24]: The FilterGroupSerializer won't leverage the `bulk_create` db method. That has to happen by swapping out the list_serializer. The item you references if for printing/viewing the data, that works fine.

File: seed/serializers/meter_readings.py [hunk 3]
```diff
@@ -4,23 +4,39 @@
...
+from django.core.exceptions import ValidationError
+from django.db.utils import IntegrityError
 from rest_framework import serializers
 
 from seed.models import MeterReading
 
 
+class MeterReadingBulkCreateUpdateSerializer(serializers.ListSerializer):
+    def create(self, validated_data):
+        print(validated_data)
```
[the remark this answers is not in the document]
Comment (nllong) [reply seed/serializers/meter_readings.py]: doh, thanks.

**Pull request merged**